eo-runtime/src/main/java/org/eolang/Main.java

```diff
@@ -187,11 +187,15 @@ private static void run(final List<String> opts) throws Exception {
             phi.attr("Δ").put(new Data.Value<>(arg));
             app.attr(0).put(phi);
         }
-        if (!new Dataized(app).take(Boolean.class)) {
-            throw new IllegalStateException(
-                "Runtime dataization failure"
-            );
-        }
+        final Object result = new Dataized(app).take();
+        Main.LOGGER.info(
+            String.format(
+                "The result of %s dataization is %s. It's value is %s",
+                app,
```

Comment:
@yasamprom here `app.toString()` will be called. Is it really what we want to happen?

Reply:
Does it work slow? What should we call instead?

Reply:
@yasamprom it will look ugly, it's a very long text. It will make your message unreadable.

Reply:
@yegor256 Let's cut `app`. We will take text after last `.`